editor/plugins/sprite_frames_editor_plugin.cpp

```diff
@@ -1023,6 +1048,20 @@ SpriteFramesEditor::SpriteFramesEditor() {
 	hbc_animlist->add_child(remove_anim);
 	remove_anim->connect("pressed", this, "_animation_remove");
 
+	toggle_anim_search_box = memnew(ToolButton);
+	toggle_anim_search_box->set_toggle_mode(true);
+	toggle_anim_search_box->set_tooltip(TTR("Show animations search box"));
+	toggle_anim_search_box->set_h_size_flags(SIZE_EXPAND | SIZE_SHRINK_END);
+	hbc_animlist->add_child(toggle_anim_search_box);
+	toggle_anim_search_box->connect("pressed", this, "_animation_toggle_search_box_visibility");
+
+	anim_search_box = memnew(LineEdit);
+	sub_vb->add_child(anim_search_box);
+	anim_search_box->set_h_size_flags(SIZE_EXPAND_FILL);
+	anim_search_box->set_placeholder(TTR("Search animations"));
```

Comment:
For consistency with other filter boxes such as "Filter nodes":

```suggestion
	anim_search_box->set_placeholder(TTR("Filter animations"));
```

Reply:
Done. Shouldn't I also for consistency change `search`->`filter` in the code?